### PR TITLE
Use relative links where possible

### PR DESCRIPTION
**Pull request by kkoci**

### CODE REVIEW

**Review by Eric-Arellano — COMMENTED** [review]
Thank you! You'll want to revert the changes to use `quantum.ibm.com` because that's not safe to do, such as in the .github template. This PR should solely have the changes to use relative links.

Also you have a merge conflict with `main`. You can use `git pull upstream main`.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ blank_issues_enabled: true
 contact_links:
   - name: Qiskit API feedback
     url: https://github.com/Qiskit/qiskit/issues/new/choose
-    about: Open an issue in the Qiskit repository for the docs found at https://docs.quantum-computing.ibm.com/api/qiskit.
+    about: Open an issue in the Qiskit repository for the docs found at https://docs.quantum.ibm.com/api/qiskit.
```
Comment (Eric-Arellano) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Bump that this change is bad in this PR. You split out your PR into two: one that solely changes the relevant URLs to use relative links, then a follow up to switch `quantum-computing` to `quantum`. This change is not ready and should be removed from this PR and only be in your follow up PR.

File: docs/api/migration-guides/qiskit-runtime-examples.mdx [hunk 2]
```diff
@@ -534,7 +534,7 @@ options that do not have a legacy alternative to migrate from, but can
 help improve your performance and results. For more information, refer
 to the following:
 
--   [Error mitigation tutorial](https://learning.quantum-computing.ibm.com/tutorial/error-suppression-and-error-mitigation-with-qiskit-runtime)
+-   [Error mitigation tutorial](../tutorial/error-suppression-and-error-mitigation-with-qiskit-runtime)
```
Comment (Eric-Arellano) [inline docs/api/migration-guides/qiskit-runtime-examples.mdx:537]: This is a bad relative link. Relative links only work if the site is `docs.quantum-computing`, not `learning.quantum-computing`.

Comment (Eric-Arellano) [reply docs/api/migration-guides/qiskit-runtime-examples.mdx:537]: There are a couple more instances of this in the PR.

File: docs/api/qiskit-ibm-provider/release-notes.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ in_page_toc_max_heading_level: 2
 
 ### Upgrade Notes
 
-*   Job error messages now include the error code. Error codes can be found in [https://quantum-computing.ibm.com/lab/docs/iql/manage/errors](https://quantum-computing.ibm.com/lab/docs/iql/manage/errors).
+*   Job error messages now include the error code. Error codes can be found in [../lab/docs/iql/manage/errors](../lab/docs/iql/manage/errors).
```
Comment (Eric-Arellano) [inline docs/api/qiskit-ibm-provider/release-notes.md:71]: This is also a bad type of relative link. The site doesn't start with `docs.quantum-computing`.

File: docs/api/migration-guides/qiskit-runtime-examples.mdx (hunk 2 above)
Comment (kkoci) [reply docs/api/migration-guides/qiskit-runtime-examples.mdx:537]: Oh, yes, sorry, had to clarify that, I was scratching my head because of that difference, good to know that, Thank You

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (kkoci) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Now I get it, sorry, thank You

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Qiskit docs
 
-The documentation content home for https://docs.quantum-computing.ibm.com. Note this repo does not contain content for https://learning.quantum-computing.ibm.com/ or https://qiskit.org. Refer to https://github.com/Qiskit/qiskit to make changes to the docs at https://qiskit.org/documentation.
```
Comment (Eric-Arellano) [inline README.md:3]: Bad change

File: README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ This is the quickest, easiest, and most helpful way to contribute to this projec
 
 - For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose).
 - For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose).
-- For problems you've found in any other section of [docs](https://docs.quantum-computing.ibm.com), open a content bug issue [here](https://github.com/Qiskit/documentation/issues/new/choose).
```
Comment (Eric-Arellano) [inline README.md:19]: Bad

File: README.md [hunk 6]
```diff
@@ -75,7 +75,7 @@ npm install
 
 Run `./start` in your terminal, then open http://localhost:3000/start in your browser.
 
-The local preview does not include the initial index page and the top nav bar from docs.quantum-computing.ibm.com. Therefore, you must directly navigate in the URL to the folder that you want:
```
Comment (Eric-Arellano) [inline README.md:78]: Bad

File: docs/api/qiskit/release-notes/0.11.md [hunk 7]
```diff
@@ -47,7 +47,7 @@ No Change
 
 ### IBM Q Provider 0.3
 
-The `IBMQProvider` has been updated in order to default to use the new [IBM Q Experience v2](https://quantum-computing.ibm.com). Accessing the legacy IBM Q Experience v1 and QConsole will still be supported during the 0.3.x line until its final deprecation one month from the release. It is encouraged to update to the new IBM Q Experience to take advantage of the new functionality and features.
```
Comment (Eric-Arellano) [inline docs/api/qiskit/release-notes/0.11.md:50]: Bad

File: docs/api/qiskit/release-notes/0.11.md [hunk 8]
```diff
@@ -62,7 +62,7 @@ IBMQ.update_account()
 For more complex cases or fine tuning your configuration, the following methods are available:
 
 *   the `IBMQ.delete_accounts()` can be used for resetting your configuration file.
-*   the `IBMQ.save_account('MY_TOKEN')` method can be used for saving your credentials, following the instructions in the [IBM Q Experience v2](https://quantum-computing.ibm.com) account page.
```
Comment (Eric-Arellano) [inline docs/api/qiskit/release-notes/0.11.md:65]: Bad

File: docs/run/estimate-job-run-time.mdx [hunk 9]
```diff
@@ -60,5 +60,5 @@ Output:
 
 You can view the estimated usage (how much quantum time the job will take to run) in two places on IBM Quantum Platform:
 
-- On the [Jobs table](https://quantum-computing.ibm.com/jobs) in the Usage column.  From the Home page, click *View all* on the Recent jobs table.  The Usage column shows the estimated usage for pending jobs, or actual usage for completed jobs.
```
Comment (Eric-Arellano) [inline docs/run/estimate-job-run-time.mdx:63]: Bad

File: docs/run/estimate-job-run-time.mdx [hunk 10]
```diff
@@ -60,5 +60,5 @@ Output:
 
 You can view the estimated usage (how much quantum time the job will take to run) in two places on IBM Quantum Platform:
 
-- On the [Jobs table](https://quantum-computing.ibm.com/jobs) in the Usage column.  From the Home page, click *View all* on the Recent jobs table.  The Usage column shows the estimated usage for pending jobs, or actual usage for completed jobs. 
-- On the job's details page. From the [Dashboard](https://quantum-computing.ibm.com/) or [Jobs table](https://quantum-computing.ibm.com/jobs), click the job ID to open the job details page.  The estimated usage is shown in the Status Timeline.
```
Comment (Eric-Arellano) [inline docs/run/estimate-job-run-time.mdx:64]: Bad

File: docs/run/fair-share-queue.mdx [hunk 11]
```diff
@@ -60,4 +60,4 @@ As described above, all jobs submitted to the scheduler through the same project
 
 A wait-time estimate is provided through IBM Quantum Platform and via Qiskit. The computed time is the result of a scheduling simulation that predicts one possible execution pattern, given the current fair-share ordering of all the jobs waiting for that system and the approximate runtime of each job. The dynamic nature of the fair-share algorithm means that this estimated time is not fixed and can vary, sometimes dramatically. This wait time is also subject to limitations inherent in estimating the execution time for Qiskit Runtime jobs. For these jobs, where an accurate estimation of time is not feasible, the maximum allowed runtime is used as a proxy. In practice, this means that the duration for a Qiskit Runtime job can be over-estimated by up to eight hours, the maximum allowed Qiskit Runtime job duration for Premium Plan users.
 
-The job’s position in the queue is listed in the **Queue position** column on the [Jobs page](https://quantum-computing.ibm.com/jobs).
```
Comment (Eric-Arellano) [inline docs/run/fair-share-queue.mdx:63]: Bad

**Review by Eric-Arellano — COMMENTED** [review]
Hey, there are still a bunch of bad changes. I didn't go through them all. But reminder you should only change URLs that start with `docs.`.

File: docs/run/fair-share-queue.mdx (hunk 11 above)
Comment (kkoci) [reply docs/run/fair-share-queue.mdx:63]: Wow, I'm so sorry, I'm new to this, will take care of it

File: README.md (hunk 4 above)
Comment (kkoci) [reply README.md:3]: `docs.` to relative urls, otherwise leave as they are

Comment (kkoci) [reply README.md]: But, in this case it should stay as quantum-computing, since it isn't safe, and also this PR will only touch relative ones, sorry, I got a bit confused, and left some quantum.ibm.com over there

File: docs/api/qiskit/release-notes/0.11.md (hunk 8 above)
Comment (kkoci) [reply docs/api/qiskit/release-notes/0.11.md:65]: This is just the original? Doesn't start with `docs.` so it is as it was, what should this be in that case? It's just `quantum-computing` without any subdomain on it

Comment (Eric-Arellano) [reply docs/api/qiskit/release-notes/0.11.md:65]: It shouldn't have any change at all in this PR because this PR is only about changing to relative links when possible, not changing `quantum-computing` to `quantum`. That's so that we can land your PR right now, rather than needing to wait until the URL changes. Does that make sense? 

This applies to most links in this PR.

File: README.md [hunk 12]
```diff
@@ -176,4 +176,4 @@ To check that formatting is valid without actually making changes, run `npm run
 2. Determine the current version of the published stable documentation, e.g. at https://github.com/Qiskit/qiskit/releases
 3. Run `npm run gen-api -- -p <pkg-name> -v <version>`, e.g. `npm run gen-api -- -p qiskit -v 0.45.0`
 
-For `qiskit`, you can also regenerate historical versions by adding `--historical` to the arguments. This is useful when we improve our generation script.
+For `qiskit`, you can also regenerate historical versions by adding `--historical` to the arguments. This is useful when we improve our generation script.
```
Comment (Eric-Arellano) [inline README.md:179]: Nit, it's better to avoid changing files with unrelated changes. I suspect the issue is if you're deleting the blank line on line 180. If so, adding that back will fix this.

PR tip: I like to review my own PRs and double check that every change I'm making is intentional.

File: docs/run/instances.mdx [hunk 13]
```diff
@@ -22,7 +22,7 @@ To see the instances to which you have access, look at the bottom of your [Accou
 
 ## Find your instances
 
-You can view a list of your instances on your [account settings page](https://quantum-computing.ibm.com/account), or you can use [the `instances()` method](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#instances).
+You can view a list of your instances on your [account settings page](https://quantum-computing.ibm.com/account), or you can use [the `instances()` method](https://quantum-computing.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#instances).
```
Comment (Eric-Arellano) [inline docs/run/instances.mdx:25]: This is a bad change. The original link was good!

File: docs/run/instances.mdx [hunk 14]
```diff
@@ -43,7 +43,7 @@ When you execute a task using an IBM Quantum service (for example, sending circu
 
 ## Open plan
 
-By default, users who sign up for an IBM Quantum account are assigned to the Open plan and the Open plan's instance, `ibm-q/open/main`. To guarantee that everyone can use the IBM Quantum systems allocated to the plan fairly, **an individual can have no more than three jobs running and/or in the queue (across all systems) at the same time.** Submitting more than three jobs at a time will return error [#3458](../errors#3458), and additional jobs will be canceled.
+By default, users who sign up for an IBM Quantum account are assigned to the Open plan and the Open plan's instance, `ibm-q/open/main`. To guarantee that everyone can use the IBM Quantum systems allocated to the plan fairly, **an individual can have no more than three jobs running and/or in the queue (across all systems) at the same time.** Submitting more than three jobs at a time will return error [#3458](https://quantum-computing.ibm.com/errors#3458), and additional jobs will be canceled.
```
Comment (Eric-Arellano) [inline docs/run/instances.mdx:46]: Ditto that the original was good. This is a "synthetic file" that we add to the docs when deploying them, even though it's not in this repository. See https://docs.quantum-computing.ibm.com/errors

File: docs/run/monitor-job.mdx [hunk 15]
```diff
@@ -23,10 +23,10 @@ Use the job instance to check the job status or retrieve the results by calling
 <span id="retrieve-later"></span>
 ## Retrieve job results at a later time
 
-Call `service.job(\<job\how_toID>)` to retrieve a job you previously submitted. If you don’t have the job ID, or if you want to retrieve multiple jobs at once; including jobs from retired systems, call `service.jobs()` with optional filters instead. See [QiskitRuntimeService.jobs](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#jobs).
+Call `service.job(\<job\how_toID>)` to retrieve a job you previously submitted. If you don’t have the job ID, or if you want to retrieve multiple jobs at once; including jobs from retired systems, call `service.jobs()` with optional filters instead. See [QiskitRuntimeService.jobs](https://quantum-computing.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#jobs).
```
Comment (Eric-Arellano) [inline docs/run/monitor-job.mdx:26]: The original was good

File: docs/run/monitor-job.mdx [hunk 16]
```diff
@@ -23,10 +23,10 @@ Use the job instance to check the job status or retrieve the results by calling
...
 
 <Admonition type="note">
-  service.jobs() returns only Qiskit Runtime jobs. To retrieve other jobs, use [qiskit-ibm-provider](../api/qiskit-ibm-provider/qiskit_ibm_provider.IBMBackend#ibmbackend) instead.
+  service.jobs() returns only Qiskit Runtime jobs. To retrieve other jobs, use [qiskit-ibm-provider](https://quantum-computing.ibm.com/api/qiskit-ibm-provider/qiskit_ibm_provider.IBMBackend#ibmbackend) instead.
```
Comment (Eric-Arellano) [inline docs/run/monitor-job.mdx:29]: The original was good

File: docs/run/run-jobs-in-session.mdx [hunk 17]
```diff
@@ -10,7 +10,7 @@ There are several ways to set up and use [sessions](sessions).  It is recommende
 
 ## Set up to use sessions
 
-Before starting a session, you must [set up Qiskit Runtime](../start/install) and initialize it as a service:
```
Comment (Eric-Arellano) [inline docs/run/run-jobs-in-session.mdx:13]: The original was good

File: docs/verify/simulate-with-qiskit-aer.ipynb
Comment (Eric-Arellano) [inline docs/verify/simulate-with-qiskit-aer.ipynb:1]: It's good you're improving this file, but it should use a relative link like `./building_noise_models`. It's in the same folder, so you can simply use `./` as the prefix

File: docs/verify/stabilizer-circuit-simulation.ipynb
Comment (Eric-Arellano) [inline docs/verify/stabilizer-circuit-simulation.ipynb:1]: The EfficientSU2 is broken. It should be a relative link.

**Review by Eric-Arellano — COMMENTED** [review]
Thanks! This is looking closer to what we want to merge. I appreciate you iterating on this.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (kkoci) [reply .github/ISSUE_TEMPLATE/config.yml]: Hi @Eric-Arellano I wonder if you had time to check the PR? Not trying to rush anything, just double checking :), shall I create the https://docs.quantum-computing.ibm.com to https://docs.quantum.ibm.com domains one?

Thank You

Comment (Eric-Arellano) [reply .github/ISSUE_TEMPLATE/config.yml]: Yes - did you see my review 4 hours ago below? This PR should only be focused on relative links. We're still waiting for docs.quantum.ibm.com and that would be in a follow up PR.

File: docs/run/run-jobs-in-session.mdx [hunk 18]
```diff
@@ -10,7 +10,7 @@ There are several ways to set up and use [sessions](sessions).  It is recommende
...
+Before starting a session, you must [set up Qiskit Runtime](https://quantum-computing.ibm.com/start/install) and initialize it as a service:
```
Comment (Eric-Arellano) [inline docs/run/run-jobs-in-session.mdx:13]:
```suggestion
Before starting a session, you must [set up Qiskit Runtime](../start/install) and initialize it as a service:
```

**Review by Eric-Arellano — APPROVED** [review]
Thank you! 🙏

**Comment by kkoci** [comment]
Thank You very much @Eric-Arellano , shall I create the quantum-computing to quantum PR? Any inputs for that although we discussed it already, but, just double checking, Thank you so much!
